root/release/edit/tracklist.tt

```diff
@@ -341,8 +341,16 @@
     <!-- /ko -->
 
     <div data-bind="visible: loaded() && !collapsed()">
+
+      <p>
+        <label>
+          <input id="tracks-unknown" type="checkbox" data-bind="checked: tracksUnknownToUser, enable: tracks() == ''" />
```

Comment:
I think the `enable` binding should be something like `tracks().length === 0` instead? Not sure I understand the string comparison.